.github/workflows/build.yml

```diff
@@ -0,0 +1,50 @@
+name: Build SaintCoinach
+on: [push, pull_request, workflow_dispatch]
```

Comment:
You probably want to remove the `pull_request` here and filter to master branch, or otherwise remove the release part of this workflow and make a secondary workflow for master pushes only that includes the release process in a separate release.yml or something.

Reply:
I added a condition for `refs/head/master` to the release step.